File: src/game/interstellarwar/Road.java
package game.interstellarwar;

import java.io.Serializable;

public class Road implements Serializable {
	private Planet mFrom, mTo;

	public Road (Planet from, Planet to) {
		this.mFrom = from;
		this.mTo = to;
	}

	public Planet getFrom () {
		return mFrom;
	}

	public Planet getTo () {
		return mTo;
	}

}

File: iOS/InterstellarWar/InterstellarWar/common/game/Road.swift
//
//  Road.swift
//  InterstellarWar
//
//  Created by Neczpál Ábel on 2020. 01. 30..
//  Copyright © 2020. Neczpál Ábel. All rights reserved.
//

import Foundation

public class Road {
    private var mFrom : Planet, mTo : Planet;

    init(from : Planet, to : Planet) {
        mFrom = from;
        mTo = to;
    }

    // GETTERS

    public func getFrom () -> Planet {
        return mFrom;
    }

    public func getTo () -> Planet {
        return mTo;
    }

}
